relay-redis/Cargo.toml

```diff
@@ -11,8 +11,8 @@ publish = false
 
 [dependencies]
 failure = "0.1.8"
-r2d2 = { version = "0.8.8", optional = true }
-redis = { version = "0.15.1", optional = true, features = ["cluster", "r2d2"] }
+r2d2 = { version = "0.8.10", optional = true }
+redis = { version = "0.22.1", optional = true, features = ["cluster", "r2d2"] }
```

Comment:
I assume no breaking changes.

Reply:
There are some breaking changes, yes. This is quite big jump between `0.15.1` -> `0.22.1`, but looks like there is nothing which could concern us. 

But there are lots of bug fixes as well.